libs/output-mapping/tests/Mapping/MappingFromProcessedConfigurationTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Tests\Mapping;
+
+use Keboola\OutputMapping\Mapping\MappingFromProcessedConfiguration;
+use Keboola\OutputMapping\Mapping\MappingFromRawConfigurationAndPhysicalData;
+use Keboola\OutputMapping\Mapping\MappingFromRawConfigurationAndPhysicalDataWithManifest;
+use Keboola\OutputMapping\Writer\FileItem;
+use Keboola\OutputMapping\Writer\Table\MappingDestination;
+use Keboola\OutputMapping\Writer\Table\Source\WorkspaceItemSource;
+use PHPUnit\Framework\TestCase;
+
+class MappingFromProcessedConfigurationTest extends TestCase
+{
+    private MappingFromProcessedConfiguration $mapping;
+
+    protected function setUp(): void
+    {
+        $mapping = [
+            'destination' => $this->createMock(MappingDestination::class),
+            'delimiter' => ',',
+            'enclosure' => '"',
+        ];
+
+        $sourceMock = $this->createMock(MappingFromRawConfigurationAndPhysicalData::class);
```

Comment:
to je dost podivný setupovat si mock v setup metodě pro jeden test

Reply:
jn to je... to jsem na začátku ještě zkoušel chatGPT na testy :D